### DISCUSSION TITLE
Indentation setting for spaces

### DISCUSSION THREAD

**Discussion by alxrey** [massCodeIO/massCode#353] [Ideas]
Add a way to select whether spaces or a tab should be inserted when the `Tab` key is pressed.

**Comment by Justsenger** [comment]
Bro I encountered the same issue as you and submitted this issue to the developer, but they simply closed it rudely and did not give me any response. I really feel puzzled about it.

**Reply by antonreshetov** [reply]
You opened the issue as a bug, and in the validation section you checked "Check that this is a concrete bug. For Q&A open a GitHub Discussion.". A bug is not a lack of some desired functionality. Closing an issue like this is not rude, nor is closing or not responding in general.

**Reply by Justsenger** [reply]
> You opened the issue as a bug, and in the validation section you checked "Check that this is a concrete bug. For Q&A open a GitHub Discussion.". A bug is not a lack of some desired functionality. Closing an issue like this is not rude, nor is closing or not responding in general.

So is there a solution to this problem? If it's not something that users can solve, I believe it's a bug.

**Reply by Faq** [reply]
More like feature

**Comment by browneyedsoul** [comment]
Just my two cents, but instead of opening the feature one by one, how about allowing users to create custom CSS blocks for making themes and plugins?
